src/app/core/core.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { RouterModule } from '@angular/router';
+import FiltersComponent from './components/filters/filters.component';
+import HeaderComponent from './components/header/header.component';
+import PageNotFoundComponent from './pages/page-not-found/page-not-found.component';
+
+@NgModule({
+  declarations: [
```

Comment:
ussually `core` component contains reusable stuff across applicaiton. In this case I don't see any reusablity for `Header/PageNotFound` logic, since it's separate main functionality